config/locales/server.en.yml

```diff
@@ -309,6 +308,8 @@ en:
     attributes:
       category:
         name: "Category Name"
+      topic:
+        title: 'Title'
```

Comment:
Rebased and zh_CN stuff is removed. Topic on meta shows a diff now. So rebase doesn't hurt anymore :P